### DISCUSSION TITLE
All settings on profile icon unavailable

### DISCUSSION THREAD

**Discussion by c-ole** [community/community#44602] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When clicking on profile icon in top right corner, there is an error message with no further detail. I am unable to access settings or even log out of my account from the web browser.

To troubleshoot, I have tried clearing cookies, resetting password, and logging in from a different device. I still am having the issue. 

![image](https://user-images.githubusercontent.com/122326637/212978279-d4c1b9c3-25f0-430f-82de-f6c3470503c4.png)

**Comment by radar-blip** [comment, marked as answer]
Same was happening to several people. It should be fixed now.

**Reply by c-ole** [reply]
Thanks!
